### PR TITLE
Finished outputting the timed bot commands stored in the hashmaps

### PR DESCRIPTION
**Pull request by brendansaw**
Minimum viable product (MVP) complete!

### CODE REVIEW

**Review by Derick-Chen — APPROVED** [review]
Looks good

**Review by k-lee07 — APPROVED** [review]
looks good to me

File: src/main/java/Pair.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    }
+
+    public MessageReceivedEvent getMessageEvent() {
+        return e;
+    }
+
+    public String getString() {
+        return s;
+    }
+}
```
Comment (jhuang40) [inline src/main/java/Pair.java:18]: Should have used library smh

Comment (brendansaw) [reply src/main/java/Pair.java:18]: Too bloated

**Review by jhuang40 — APPROVED** [review]

**Pull request merged**